css/components/list.css

```diff
@@ -21,3 +21,14 @@
     border-bottom: 1px solid var(--default);
     list-style: none;
 }
+
+.list-item-image{
+    object-fit: cover;
+    width: 50px;
+    height: 50px;
```

Comment:
use rems here

Reply:
Would recommend using rems to make image responsive.

Reply:
changed